### ISSUE TITLE
Auto android build

### ISSUE THREAD

**Issue by zhuiks** [ezra-project/ezra-project#229]
How do you build for Android, I couldn't find any docs.

Can it be put in Github Actions?

**Comment by tobias-klein** [comment]
Yeah, it's not properly documented at this point. I'll see whether I can add some instructions as well as look into the auto build option. You can have a peek into build.sh in ezra-project-cordova. But that'll not explain the preconditions.
